index 6d3ce93..18fe653 100644
--- a/src/operator/tensor/elemwise_binary_broadcast_op_basic.cc
+++ b/src/operator/tensor/elemwise_binary_broadcast_op_basic.cc
@@ -23,7 +23,7 @@
  * \brief CPU Implementation of basic functions for elementwise binary broadcast operator.
  */
 #include "./elemwise_unary_op.h"
-#include "./elemwise_binary_op.h"
+#include "./elemwise_binary_op-inl.h"
 #include "./elemwise_binary_broadcast_op.h"

 namespace mxnet {
index aa0850a..75ab2ae 100644
--- a/src/operator/tensor/elemwise_binary_broadcast_op_basic.cu
+++ b/src/operator/tensor/elemwise_binary_broadcast_op_basic.cu
@@ -23,7 +23,7 @@
  * \brief GPU Implementation of basic functions for elementwise binary broadcast operator.
  */
 #include "./elemwise_unary_op.h"
-#include "./elemwise_binary_op.h"
+#include "./elemwise_binary_op-inl.h"
 #include "./elemwise_binary_broadcast_op.h"

  8  
libmxnet/meta.yaml