diff --git a/src/operator/tensor/elemwise_binary_op.h b/src/operator/tensor/elemwise_binary_op.h
index 2fe3fd991..d072ab0d6 100644
--- a/src/operator/tensor/elemwise_binary_op.h
+++ b/src/operator/tensor/elemwise_binary_op.h
@@ -784,3 +784,6 @@ class ElemwiseBinaryOp : public OpBase {
 }  // namespace op
 }  // namespace mxnet
 #endif  // MXNET_OPERATOR_TENSOR_ELEMWISE_BINARY_OP_H_
+
+// Quick workaround for a linker issue.
+#include "elemwise_binary_op-inl.h"
